plugins/modules/packaging/os/snap.py

```diff
@@ -46,6 +47,14 @@
         type: str
         required: false
         default: stable
+    options:
+        description:
+            - Set options with pattern key=value or snap:key=value. If a snap name is given, the option will be applied
+              to that snap only. If the snap name is omitted, the options will be applied to all snaps. Options will
```

Comment:
```suggestion
              to that snap only. If the snap name is omitted, the options will be applied to all snaps listed in I(name). Options will
```
Or does it apply to all snaps installed in the system?